modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaClayDataSetDataProviderCheck.testjava

```diff
@@ -5,6 +5,10 @@
 
 package com.liferay.source.formatter.dependencies.upgrade;
 
+import com.liferay.frontend.taglib.clay.data.set.provider.ClayDataSetDataProvider;
+import com.liferay.frontend.taglib.clay.data.Filter;
+import com.liferay.frontend.taglib.clay.data.Pagination;
```

Comment:
@kyle-miho why did you add these imports? 
If isn't necessary to the logic, we can keep it simple and remove it, what do you think?

Reply:
@nicolas-sss the imports.txt replacements depend on this, but i think its better to remove it, so we can still perform the upgrade and fix their missing imports if they dont include these, ill just add the imports in newImports if not already present to account for this

Reply:
@nicolas-sss this is technically required since we are using imports.txt / UpgradeJavaCheck to perform a large bulk of the upgrades like you told me, we need those imports to exist otherwise it wont perform the replacements for all of its usages.